chapters/chapter1/chapter1-3.tex

```diff
@@ -93,7 +93,13 @@ \section{The Axiom of Completeness}
 		&\Rightarrow\, \frac{s}{c} = \sup A \\
 		&s = c \sup A
 	\end{align*}
-	\item If $c < 0$, $\sup(cA) = -c\inf(A)$.
+	\item If $c < 0$, \begin{align*}
```

Comment:
I'm being pretty picky, but can you make the align start on a new line?

Reply:
Sure thing